openapi/src/parts/azure_landing_zone.yaml

```diff
@@ -200,9 +223,35 @@ components:
           description: The ID of Azure resource group associated with the landing zone.
           type: string
 
+    AzureLandingZoneResourcesList:
+      type: object
+      required: [resources]
+      properties:
+        id:
+          description: An identifier of a Azure landing zone.
+          type: string
+        resources:
+          description: A list of deployed resources in a landing zone, grouped by purpose.
+          type: array
+          items:
+            $ref: '#/components/schemas/AzureLandingZoneResourcesPurposeGroup'
+
+    AzureLandingZoneResourcesPurposeGroup:
+      type: object
+      required: [purpose,deployedResources]
+      properties:
+        purpose:

```

Comment:
I don't understand what this is.

In general, is there a document where this API is specified? If not, I think there should be.

Reply:
The structure contains azure deployed resources grouped by the landing zone purpose tag. 
The grouping is useful to identify landing zone resources for the down steam landing zone scenarios.